platform/commonUI/browse/bundle.json

```diff
@@ -80,7 +80,7 @@
             },
             {
                 "key": "urlService",
-                "implementation": "services/UrlService.js",
+                "implementation": "../../general/src/services/UrlService.js",
                 "depends": [ "$location" ]
```

Comment:
Makes sense to move this JavaScript file to `general`, but we should move this extension definition to `platform/commonUI/general` as well